storey/aggregations.py

```diff
@@ -172,27 +173,44 @@ async def _emit_worker(self):
             next_emit_time = next_emit_time + seconds_to_sleep_between_emits
 
 
-class QueryAggregationByKey(AggregateByKey):
+class QueryByKey(AggregateByKey):
     """
-    Similar to to `AggregateByKey`, but this step is for serving only and does not aggregate the event.
+    Query features by name
 
-    :param aggregates: List of aggregates to apply for each event.
+    :param features: List of features to get.
+    :type features: list of string
     :param table: A Table object to aggregate the data into.
+    :type table: Table
     :param key: Key field to aggregate by, accepts either a string representing the key field or a key extracting function.
      Defaults to the key in the event's metadata. (Optional)
-    :param emit_policy: Policy indicating when the data will be emitted. Defaults to EmitEveryEvent. (Optional)
+    :type key: string or Function (Event=>object)
     :param augmentation_fn: Function that augments the features into the event's body. Defaults to updating a dict. (Optional)
-    :param enrich_with: List of attributes names from the associated storage object to be fetched and added to every event. (Optional)
+    :type augmentation_fn: Function ((Event, dict) => Event)
     :param aliases: Dictionary specifying aliases to the enriched columns, of the format `{'col_name': 'new_col_name'}`. (Optional)
+    :type aliases: dict
     """
 
-    def __init__(self, aggregates: Union[List[FieldAggregator], List[Dict[str, object]]], table: Table,
-                 key: Union[str, Callable[[Event], object], None] = None,
-                 emit_policy: Union[EmitEveryEvent, FixedWindows, SlidingWindows, EmitAfterPeriod, EmitAfterWindow,
-                                    EmitAfterMaxEvent, Dict[str, object]] = _default_emit_policy,
-                 augmentation_fn: Optional[Callable[[Event, Dict[str, object]], Event]] = None, enrich_with: Optional[List[str]] = None,
-                 aliases: Optional[Dict[str, str]] = None, **kwargs):
-        AggregateByKey.__init__(self, aggregates, table, key, emit_policy, augmentation_fn, enrich_with, aliases, **kwargs)
+    def __init__(self, features, table, key=None, augmentation_fn=None, aliases=None):
+        self._schema = table._storage._load_schema(table._container, table._table_path)
+        self._aggrs = []
+        self._enrich_cols = []
+        resolved_aggrs = {}
+        for feature in features:
+            if re.match(r".*_[a-z]+_[0-9]+[a-z]", feature):
+                parts = feature.rsplit('_', 1)
+                if parts[0] in resolved_aggrs:
+                    resolved_aggrs[parts[0]].append(parts[1])
+                else:
+                    resolved_aggrs[parts[0]] = [parts[1]]
+            else:
+                self._enrich_cols.append(feature)
+        for name, windows in resolved_aggrs.items():
+            parts = name.rsplit('_', 1)
```

Comment:
same here, better initialize with `aggregate_name, aggregate = name.rsplit('_', 1)` or something of the sort